site/src/components/MainNav.tsx

```diff
@@ -2,29 +2,14 @@ import {
   Link as NavLink,
   NavigationBar,
 } from "@cultureamp/kaizen-component-library"
-import { graphql, useStaticQuery, withPrefix } from "gatsby"
+import { withPrefix } from "gatsby"
 import * as React from "react"
 
 type MainNavProps = {
   currentPath?: string
 }
 
 const MainNav: React.SFC<MainNavProps> = ({ currentPath = "" }) => {
-  const data = useStaticQuery(graphql`
-    query MainNavComponentsQuery {
-      allMdx(filter: { fields: { slug: { regex: "^/components/" } } }) {
-        edges {
-          node {
-            fields {
-              slug
-            }
-          }
-        }
-      }
-    }
-  `)
-  const firstComponentPath = data.allMdx!.edges[0]!.node!.fields!.slug
```

Comment:
I have no idea what this was, but I'm glad it's gone!

Reply:
Before, there was no components overview, so we had to query to find out what the first component was and make a link for that. Now we just hard-code `components/overview`.